cryptodoc/src/13_pbkdf.rst

```diff
@@ -0,0 +1,17 @@
+Password Based Key Derivation Functions
+=======================================
+
+Argon2 (RFC 9106)
+-----------------
+
+Argon2 is a memory-hard function for password hashing.
+The implementation in Botan can be found in :srcref:`src/lib/pbkdf/argon2/argon2.cpp`
+and follows the standard [RFC9106]_.
+
+The maximum parallelism in Botan is more restricted than specified in [RFC9106]_.
+Botan allows a maximum of 128 threads.
+
+The maximum memory size in Botan is more restricted than specified in [RFC9106]_.
+Botan allows a maximum of :math:`2^{23}` KiB.
```

Comment:
Nit: When using powers of two, maybe also use the base unit?

```suggestion
Botan allows a maximum of :math:`2^{26}` bytes.
```

... on the other hand: the RFC uses kibibytes as the unit. So maybe it makes sense to keep it here as well. 🤔

Reply:
How to you come to 2^26 bytes? 

I slightly prefer to keep the unit as KiB. As you noted it is the unit the RFC uses, and it is also the unit used in Botan.